script.service.ltvpl/resources/lib2/enum/LICENSE

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2013, Ethan Furman.
```

Comment:
this has to be submitted as an external python script.module

Reply:
My add-on requires the [script.module.enum34] (PR 1108) module in order to run. The enum module is not included in Python 2 but is included in Python 3. This PR was closed recently because no add-on in the repository references it.

It is my intention that this version of my add-on will only be qualified to run under Leia. I will be removing the requirement for the [script.module.enum34] module for Matrix since it will be running Python 3.

Reply:
Removed from my add-on.